docs/Tutorials/CrossCompilationSetup/ArmLinuxTutorial.md

```diff
@@ -7,25 +7,25 @@ First, in a terminal upload the software to hardware platform. This is done with
 ```sh
 # For ARM 64-bit hardware
 # In: Deployment Folder
-scp -r build-artifacts/aarch64-linux/<name-of-deployment> <username>@<device-address>:deployment
+scp -r ../build-artifacts/aarch64-linux/<name-of-deployment> <username>@<device-address>:deployment
```

Comment:
It should probably call out `# In project root` above and in the other scripts as well, instead of `# In: Deployment Folder`

Reply:
Sounds good to me. I can make that update

Reply:
Done